Successfully synced application org.nativescript.plugindemo on device 192.168.59.101:5555.
JS: get series from android - rx
JS: httpLibrary com.emedinaa.sample.httplibrary.NativeWrapper@c3c111e
System.err: An uncaught Exception occurred on "main" thread.
System.err: Calling js method onClick failed
System.err: Error: android.os.NetworkOnMainThreadException
System.err:
System.err: StackTrace:
System.err: getSeriesRx(file: src/packages/httplibrary/index.android.ts:30:15)
System.err: 	at start(file: src/src/plugin-demos/httplibrary.ts:14:2)
System.err: 	at _handleEvent(file: node_modules/@nativescript/core/data/observable/index.js:230:0)
System.err: 	at notify(file: node_modules/@nativescript/core/data/observable/index.js:216:0)
System.err: 	at _emit(file: node_modules/@nativescript/core/data/observable/index.js:256:0)
System.err: 	at ClickListenerImpl.onClick(file: node_modules/@nativescript/core/ui/button/index.android.js:28:0)
System.err: 	at com.tns.Runtime.callJSMethodNative(Native Method)
System.err: 	at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err: 	at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err: 	at com.tns.gen.java.lang.Object_vendor_34075_28_ClickListenerImpl.onClick(Object_vendor_34075_28_ClickListenerImpl.java:18)
System.err: 	at android.view.View.performClick(View.java:7259)
System.err: 	at android.view.View.performClickInternal(View.java:7236)
System.err: 	at android.view.View.access$3600(View.java:801)
System.err: 	at android.view.View$PerformClick.run(View.java:27892)
System.err: 	at android.os.Handler.handleCallback(Handler.java:883)
System.err: 	at android.os.Handler.dispatchMessage(Handler.java:100)
System.err: 	at android.os.Looper.loop(Looper.java:214)
System.err: 	at android.app.ActivityThread.main(ActivityThread.java:7356)
System.err: 	at java.lang.reflect.Method.invoke(Native Method)
System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
System.err: Caused by: android.os.NetworkOnMainThreadException
System.err: 	at android.os.StrictMode$AndroidBlockGuardPolicy.onNetwork(StrictMode.java:1565)
System.err: 	at java.net.Inet6AddressImpl.lookupHostByName(Inet6AddressImpl.java:115)
System.err: 	at java.net.Inet6AddressImpl.lookupAllHostAddr(Inet6AddressImpl.java:103)
System.err: 	at java.net.InetAddress.getAllByName(InetAddress.java:1152)
System.err: 	at okhttp3.Dns$Companion$DnsSystem.lookup(Dns.kt:49)
System.err: 	at okhttp3.internal.connection.RouteSelector.resetNextInetSocketAddress(RouteSelector.kt:164)
System.err: 	at okhttp3.internal.connection.RouteSelector.nextProxy(RouteSelector.kt:129)
System.err: 	at okhttp3.internal.connection.RouteSelector.next(RouteSelector.kt:71)
System.err: 	at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.kt:197)
System.err: 	at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.kt:107)
System.err: 	at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.kt:75)
System.err: 	at okhttp3.internal.connection.RealCall.initExchange$okhttp(RealCall.kt:245)
System.err: 	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:32)
System.err: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:100)
System.err: 	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.kt:82)
System.err: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:100)
System.err: 	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.kt:83)
System.err: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:100)
System.err: 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:74)
System.err: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:100)
System.err: 	at b.a.intercept(Unknown Source:59)
System.err: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:100)
System.err: 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:197)
System.err: 	at okhttp3.internal.connection.RealCall.execute(RealCall.kt:148)
System.err: 	at a.a.getRx(Unknown Source:2)
System.err: 	at com.emedinaa.sample.httplibrary.NativeWrapper.getRx(Unknown Source:12)
System.err: 	... 18 more
